contracts/MarketCollateralPool.sol

```diff
@@ -112,9 +112,11 @@ contract MarketCollateralPool is Ownable {
     /// @param depositAmount qty of ERC20 tokens to deposit to the smart contract to cover open orders and collateral
     function depositTokensForTrading(address collateralTokenAddress, uint256 depositAmount) external {
         uint256 balanceAfterDeposit = tokenAddressToAccountBalance[collateralTokenAddress][msg.sender].add(depositAmount);
+        uint256 balanceBeforeDeposit = tokenAddressToAccountBalance[collateralTokenAddress][msg.sender];
+        int256 changeInBalance = int256(balanceAfterDeposit).subtract(int256(balanceBeforeDeposit));
```

Comment:
Shouldn't this amount be the exact same amount as `depositAmount` represented as an integer?

Reply:
Yes, @pelsasser it should be the same! I already mentioned that in the comments I made while explaining my solution before I started coding, I just did not get a clear direction about it from your replies.